application/src/ext/java/org/opentripplanner/ext/carpooling/README.md

```diff
@@ -0,0 +1,468 @@
+# Carpooling Extension for OpenTripPlanner
```

Comment:
For the sandbox modules, you should (also) have a document like https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/doc/user/sandbox/Flex.md. For constructing it, you can use a templating https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/doc/templates/Flex.md